jenkins/ci.opensuse.org/openstack-devstack.yaml

```diff
@@ -42,15 +43,15 @@
           ssh root@cleanvm "bash -x ~/qa_devstack.sh {ipversion}"
           ret=$?
           if [ "$ret" != 0 ] ; then
-              virsh shutdown cleanvm
+              $ci1_ssh virsh shutdown cleanvm
               # wait for clean shutdown
-              n=20 ; while [[ $n > 0 ]] && virsh list |grep cleanvm.*running ; do sleep 2 ; n=$(($n-1)) ; done
-              virsh destroy cleanvm
-              find /mnt/cleanvmbackup -mtime +5 -type f | xargs --no-run-if-empty rm
+              n=20 ; while [[ $n > 0 ]] && $ci1_ssh "virsh list |grep cleanvm.*running" ; do sleep 2 ; n=$(($n-1)) ; done
+              $ci1_ssh virsh destroy cleanvm
+              $ci1_ssh "find /mnt/cleanvmbackup -mtime +5 -type f | xargs --no-run-if-empty rm"
               # backup /dev/vg0/cleanvm disk image
               file=/mnt/cleanvmbackup/$BUILD_NUMBER-devstack.raw.gz
-              time gzip -c1 /dev/vg0/cleanvm > $file
-              du $file
+              $ci1_ssh "time gzip -c1 /dev/vg0/cleanvm > $file"
```

Comment:
do we even use these image backups sometimes? Otherwise we could save time here and also skip the 'clean shutdown' above.

Reply:
I have no idea... I just updated the existing scripting to continue to work in the new environment. Perhaps @dirkmueller has a better idea?

Reply:
I think this could be changed in a future PR if it needs to be revisited